### ISSUE TITLE
mkchain broken

### ISSUE THREAD

**Issue by nicolasochem** [oxheadalpha/tezos-k8s#315]
maybe broken by #313?

```
[venv] nochem@peck ~/workspace/tezos-k8s () $ mkchain nico
Generating vanity chain id
Checking for docker image registry.gitlab.com/tezos/flextesa:01e3f596-run
Generating keys for account tezos-baking-node-0
Checking for docker image tezos/tezos:v10-release
Wrote chain creation constants to /home/nochem/workspace/tezos-k8s/nico_values.yaml
[venv] nochem@peck ~/workspace/tezos-k8s () $ helm install nico -f nico_values.yaml  charts/tezos --namespace nico --create-namespace
Error: YAML parse error on tezos-chain/templates/nodes.yaml: error converting YAML to JSON: yaml: line 17: did not find expected key
```

**Comment by nicolasochem** [comment]
yes, it's missing `labels`

**Comment by harryttd** [comment]
This should automatically be fixed by #314

**Comment by nicolasochem** [comment]
You are right. That's why submodules are good. Release should never be in the way of anything 😉
